src/tasks/S3Upload/TaskOperations.ts

```diff
@@ -171,11 +172,11 @@ export class TaskOperations {
             if (!stats.isDirectory()) {
                 const fileBuffer = fs.createReadStream(matchedFile)
                 try {
-                    let contentType: string | undefined
+                    let contentType: string | false | undefined
```

Comment:
I don't think this can return undefined

Reply:
Yeah, I think I was trying to cover for the `this.taskParameters.contentType`.